src/gui/previewselectdialog.h

```diff
@@ -60,6 +60,11 @@ class PreviewSelectDialog final : public QDialog
         NB_COLUMNS
     };
 
+    enum Roles
```

Comment:
It isn't supposed to be used outside of `PreviewSelectDialog` implementation so I would move it in .cpp.

Reply:
https://github.com/qbittorrent/qBittorrent/blob/a81df9d43501dcf4e7ea05a921350029d9e8855f/src/gui/previewselectdialog.cpp#L51-L54
moved to .cpp to reduce scope